hystrix/metric_collector/metric_collector.go

```diff
@@ -45,6 +45,8 @@ func (m *metricCollectorRegistry) Register(initMetricCollector func(string) Metr
 type MetricCollector interface {
 	// IncrementAttempts increments the number of updates.
 	IncrementAttempts()
+	// IncrementQueueSize increments the number of elements in the queue.
+	IncrementQueueSize()
```

Comment:
"queue size/length" does not seem like an accurate name since this is better stated as "rate at which executions were queued", or "number of queued events over a time window"

Reply:
I have changed the name to `IncrementQueuedItem`. I tried to keep it in sync with other function names like `IncrementAttempts`. Let me know if you think of some other function name to be more appropriate.